doc/dev/GOVERNANCE.md

```diff
@@ -1,18 +1,49 @@
+This document provides the governance policy for F3D (the "Project"). [Every person](./MAINTAINERS_AND_CONTRIBUTORS.md) responsible for its maintenance (the "Maintainers") agrees to this policy and to abide by all Project policies, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [MAINTAINERS_AND_CONTRIBUTORS.md file](./MAINTAINERS.md).
 
 ## 1. Roles
 
 This Project includes the following roles.
 
 ### 1.1. Maintainers
 
-Maintainers are responsible for organizing activities around developing, maintaining, and updating the Project. Maintainers are also responsible for determining consensus. The Project may add or remove Maintainers by affirmative vote of 2/3 of all Maintainers.
+Maintainers are responsible for organizing activities around developing, maintaining, and updating the Project. Maintainers are responsible of upholding the [Code of Conduct](../../CODE_OF_CONDUCT.md). Maintainers are also responsible for determining consensus for technical or community matters. The Project may add or remove Maintainers by affirmative vote of 2/3 of all Maintainers. Maintainers are de-facto moderators. Maintainers are part of the f3d-app github organization with a "owner" role and listed in the [MAINTAINERS_AND_CONTRIBUTORS.md file](./MAINTAINERS_AND_CONTRIBUTORS.md).
 
-### 1.2. Contributors
+Maintainers is an important role and as such, will be evaluated by the community yearly and exit interview may be conducted in case of a Maintainer leaving the role.
 
-Contributors are those who have made contributions to the Project.
+Any Returning Contributor interested to become a Maintainer should just reach out to existing Maintainers for consideration.
+
+### 1.1. Moderators
+
+Moderators are responsible of upholding the [Code of Conduct](../../CODE_OF_CONDUCT.md). Moderators are responsible for determining consensus for community matters. The Project may add or remove Maintainers by affirmative vote of 2/3 of all Maintainers.
+
+Anyone interested to become a moderators should just reach out to existing Maintainers for consideration.
+
+### 1.2. Returning Contributors
+
+Contributors are those who have made multiple contributions to the Project and want to be acknowledged as such. Returning Contributors are also very much welcome to share their ideas and feedbacks about the development and maintenance of F3D.
+
+To become a Returning Contributor, one need to have merged at least two pull requests into the F3D repositories. Returning Contributors are part of the f3d-app github organization with a "member" role and are listed in the [MAINTAINERS_AND_CONTRIBUTORS.md file](./MAINTAINERS_AND_CONTRIBUTORS.md).
```

Comment:
> and are listed in the MAINTAINERS_AND_CONTRIBUTORS.md file

I think some people may want to refuse it for any personal reason,

Reply:
> Contributors are those who have made multiple contributions to the Project and want to be acknowledged as such

Its in this sentence in my opinion. Contributors that do not want to be listed in this file are just contributors, not returning contributors, as simple as that :)

Reply:
Yes but what about returning contributors wanting to have some privileges, but refusing to be listed publicly?